tests/spec/core/inlines-spec.js

```diff
@@ -224,6 +224,44 @@ describe("Core - Inlines", () => {
     expect(badOne.textContent).toBe("[[[#does-not-exist]]]");
   });
 
+  it("supports bikeshed-like `[[[spec/path#fragment]]]`section links", async () => {
+    const body = `
+      <p id="output">
+        [[[!HTML/webstorage.html#user-tracking]]]
+        [[[?HTML/webstorage.html#introduction-15]]]
+        [[[HTML/webstorage.html#disk-space-2]]]
+      </p>
+    `;
+
+    const doc = await makeRSDoc(makeStandardOps(null, body));
+    const anchors = doc.querySelectorAll("#output a");
+
+    expect(anchors.length).toBe(3);
+
+    const [normativeAnchor, informativeAnchor, generalAnchor] = anchors;
+
+    expect(normativeAnchor.textContent).toBe(
+      "[HTML/webstorage.html#user-tracking]"
```

Comment:
This doesn't look right.... I was expecting just "[HTML]" as we never show fragments or the document to end-users (the link itself is fine though). Do you have a link to a spec that does this or the BikeShed documentation?

Reply:
Found the docs... commented here https://github.com/w3c/respec/pull/2724#issuecomment-577488110

We would need to expand these out so: 

```
[[css-flexbox-1#auto-margins]]
```

Becomes:

```HTML
<a href="https://drafts.csswg.org/css-flexbox-1/#auto-margins">CSS Flexbox 1 §8.1 Aligning with auto margins</a>
```

@sidvishnoi, do you know if Shepherd has this data?

Reply:
fwiw I tried
`bikeshed refs --text=css`
`bikeshed refs --text=flexbox`
`bikeshed refs --text=auto-margins`
in console and didn't get any results.

Reply:
no idea how Bikeshed works :) however, BikeShed pulls data from Shepherd, so in theory this stuff is in the Shepherd DB. If that’s true, we may be able to do this, but will require significant work as we will need to also add support to the respec.org end-points.